daemon/api.go

```diff
@@ -732,6 +732,7 @@ func snapshotMany(inst *snapInstruction, st *state.State) (*snapInstructionResul
 }
 
 type snapActionFunc func(*snapInstruction, *state.State) (string, []*state.TaskSet, error)
+type snapsActionFunc func(*snapInstruction, *state.State) (*snapInstructionResult, error)
```

Comment:
could this be named `manySnapsActionFunc` to distinguish it visually, and also would fit nicely with `dispatchMany() (op manySnapsActionFunc)` ?

Reply:
to be clear I picked the name to follow postSnap vs postSnaps (which maybe is problematic in itself).  I suppose we could as well rename postSnaps to postSnapMany?

Reply:
Yeah, that would be nice too.

Reply:
ok, I think I'll do both in a the follow-up though, where I will move current postSnaps to api_snaps.go